uast/diff/diff.go

```diff
@@ -0,0 +1,287 @@
+package diff
+
+import (
+	"fmt"
+	"github.com/heetch/lapjv"
+	"gopkg.in/bblfsh/sdk.v2/uast/nodes"
+)
+
+type decisionType interface {
+	isDecisionType()
```

Comment:
`isDecisionType()` can be removed, since `cost()` already prevents external packages from implementing it (assuming other types in this package has no such method).